bolsas.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<head>
+<html lang="pt-br">
+<header>
```

Comment:
Aqui deve ser uma tag `head` mesmo, `header` só deve ser usado dentro do `body`
```suggestion
<head>
```

Reply:
Ok, trocando. Sempre tive essa duvida